test/lib/jdk/test/lib/process/OutputBuffer.java

```diff
@@ -146,14 +147,18 @@ public String getStderr() {
 
     @Override
     public int getExitValue() {
+      Integer exitCode = this.processExitCode;
+      if (exitCode != null) {
+        return exitCode;
+      }
       try {
           logProgress("Waiting for completion");
           boolean aborted = true;
           try {
-              int result = p.waitFor();
+              this.processExitCode = exitCode = p.waitFor();
```

Comment:
According to the `waitFor` javadocs it returns the "exit value" and this function is named `getExitValue()`. I propose that we rename `processExitCode` to `exitValue` (alt. `processExitValue`).

Reply:
With the earlier suggestion, this would become:
```
exitValue = p.waitFor();
```

Reply:
You are right - exitValue instead of exitCode would be appropriate. I've updated the PR to implement this suggestion.